common/static/common/js/components/views/feedback.js

```diff
@@ -4,7 +4,8 @@
             "underscore",
             "underscore.string",
             "backbone",
-            "text!common/templates/components/system-feedback.underscore"],
+            "text!common/templates/components/system-feedback.underscore",
+            "jquery.ui.selectable"],
```

Comment:
I didn't realize that you'd bundled it separately from JQuery UI itself. Will it be a problem to load this and the full JQuery UI bundle on the same page?